src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements Area {
```

Comment:
```suggestion
public class Circle extends Figure {
```

if class Figure implements some interface, then all it's heirs will implement this interface automatically